<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDM_Lab1_Graphs</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="operation">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Operations</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Incidence ---&gt; Adjacency</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Incidence ---&gt; Adjacency (not-oriented)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Adjacency --&gt; Incidence </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Adjacency --&gt; Incidence (not-oriented)</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="y">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Y"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="x">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="X"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="firstMatrix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="secondMatrix">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="aboveFirstMatrix">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="aboveSecondMatrix">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="build">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="oriented_radio">
       <property name="text">
        <string>Oriented</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="notoriented_radio">
       <property name="text">
        <string>Not-oriented</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>206</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>the number of vertices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="vertice"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
